src/views/document/XreportView.vue

```diff
@@ -23,6 +23,10 @@
           <field-view :document="document" :field="fields.nb_impacted" />
```

Comment:
should remove summary

Reply:
activity rate and autonomy are now displayed, not restricted to data analysis?

Reply:
the same question for supervision and qualification.

Reply:
I chose not to modify the data model,
I would suggest the following behaviour:
existing summaries will still be displayed, but the users cannot enter new ones via the form.

Reply:
> activity rate, autonomy, supervision and qualification are now displayed, not restricted to data analysis?

Is it handled by the API? If yes, fields will be empty for non-moderators. It no, it should :)

> existing summaries will still be displayed, but the users cannot enter new ones via the form.

Ok for me.